docs/source/support/release-notes.rst

```diff
@@ -9,11 +9,15 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+0.12.7 — 2019-04-23
```

Comment:
```suggestion
0.12.8 — 2019-04-23
```

Reply:
such a simple task and still screwed it up :D